src/shop-app.html

```diff
@@ -441,7 +441,7 @@
             Polymer.importHref(this.resolveUrl('lazy-resources.html'), () => {
               // Register service worker if supported.
               if ('serviceWorker' in navigator) {
-                navigator.serviceWorker.register('/service-worker.js');
+                navigator.serviceWorker.register('service-worker.js', {scope: '/'});
```

Comment:
Good question, @cdata . In our proposed scheme for multiple builds / differential serving, each individual build has its own service worker, presumably partially auto-generated by the build process (as is the case with our toolchain).

There is a small bit of server- or network-resident logic in this scheme that chooses one of the individual builds based on the visiting user agent, and based on that logic the service worker associated with that build is the one that gets registered.

In theory, any special SW logic you had for specific purposes would be essentially unaffected by our build-switching scheme; it would just need to find its way into each of the individual service workers in the final file layout.

Thoughts?